src/Nethermind/Nethermind.JsonRpc.Test/Modules/Eth/EthRpcModuleTests.cs

```diff
@@ -1082,6 +1082,38 @@ public static void Should_handle_gasCap_as_max_if_null_or_zero(long? gasCap)
         Assert.AreEqual(long.MaxValue, rpcTx.Gas, "Gas must be set to max if gasCap is null or 0");
     }
 
+    [Test]
+    public async Task eth_getBlockByNumber_should()
```

Comment:
@rubo eth_getBlockByNumber_should should what? 

Looks like wee need such test, but with non empty withdrawals

Reply:
I have no idea "should what" 🙂

Reply:
The test name is weird. Better names:
eth_getBlockByNumber_should_return_expected_result
eth_getBlockByNumber_should_return_withdrawals_correctly